src/main.rs

```diff
@@ -49,6 +53,9 @@ struct Args {
     skip_img_copy: bool,
     #[arg(long)]
     output_path: Option<String>,
+    #[arg(long, default_value_t = false)]
+    protection: bool,
```

Comment:
Like with the params above, I think we can simplify these down to a single argument. Instead of both a bool and a type, we can have the user specify a `--protection-type` and trust that if it's set, we want protection, and if not, we don't want protection.